core/bin/zksync_server/src/node_builder.rs

```diff
@@ -703,15 +714,15 @@ impl MainNodeBuilder {
                 Component::HttpApi => {
                     self = self
                         .add_l1_gas_layer()?
-                        .add_tx_sender_layer()?
+                        .add_tx_sender_layer(deny_list_enabled)?
```

Comment:
if you run the command "zk server run --components "tree,eth,state_keeper,housekeeper,proof_data_handler,deny_list" would that work? 

It seems the order matter does not seem ideal. I feel in that case it would be better to turn denylist into a separate flag.

Alternatively I think there may be way to discover and wire components, but it would be more complex to implement

Reply:
Because all components will be sorted before add layers and `HttpApi` and `WsApi` component are lowest priority, see
https://github.com/cronos-labs/cronos-zkevm/pull/79/files/5db68cb518793fd88c51657a2656e484b07a4bda#diff-0ae53db055121ab669cf03a36842e15fe6bb898c316f02e679c1ff123e2dbceeL685

Reply:
ok